packages/components/addon/components/hds/form/fieldset/index.hbs

```diff
@@ -0,0 +1,32 @@
+<fieldset class={{this.classNames}} ...attributes aria-describedby={{this.ariaDescribedBy}}>
```

Comment:
maybe we can have the same approach as in the `field` and declare a container that does nothing more that yielding its content?